integration/helper_test.go

```diff
@@ -73,6 +74,16 @@ func gpbackupHelper(helperPath string, args ...string) *exec.Cmd {
 	return command
 }
 
+func gpbackupHelperRestoreNoSingle(helperPath string, args ...string) *exec.Cmd {
+
+	args = append([]string{"--toc-file", tocFile, "--pipe-file", pipeFile,
```

Comment:
Usually gprestore will call helper either if [single data file used, or resize cluster is required](https://github.com/arenadata/gpbackup/blob/21ba1a1c96c8ed2ee5cd27b201b0b2c2547eed2e/restore/data.go#L207)
So, starting helper with non-single data file and without resize keys (" --resize-cluster --orig-seg-count %d --dest-seg-count %d") we leave one of the major use cases uncovered.

Reply:
test rewritten